network/src/commonMain/kotlin/com/wire/kalium/network/api/asset/AssetApiImpl.kt

```diff
@@ -150,3 +184,5 @@ class StreamAssetContent(
         channel.close()
     }
 }
+
+private const val BUFFER_SIZE = 1024 * 4L
```

Comment:
Why not 8KB?

Reply:
Just to avoid the small chance to use extra memory, but is a valid point, we could make it 8KB and tweak down if failures, would be more efficient.

Also, I'll check this uploading, I tried with a 65 MB and worked, but will try with a bigger one and more edge cases =)

Reply:
Just tested, with several a prev. Downloaded asset of 91 MB, and the app could not handle :(  So I believe we need to do the same for upload asset, namely for encryption, use a form of output/input stream instead of `ByteArray`. We can sync tomorrow to have a better an idea on how to do it, maybe I can raise another PR.